LeanMapperQuery/IQuery.php

```diff
@@ -23,4 +23,17 @@
 	 * @return Fluent
 	 */
 	public function applyQuery(Fluent $fluent, IMapper $mapper);
+
+	/**
```

Comment:
Tohle způsobí BC break, ale nejspíš nestojí za to kvůli tomu vytvářet nové rozhraní, protože těch, kteří si vytvořili vlastní implementaci `IQuery` bude asi minimum (dost možná nikdo).

Reply:
Jj, to mě taky trápí, ale právě jsem si říkal, že to asi není potřeba řešit, pochybuji, že si někdo vytvářel vlastní implementaci. Takže bych řekl, že bude stačit, když se na to upozorní v poznámkách k vydání.